bee-crypto/tests/batched_curlp.rs

```diff
@@ -38,7 +38,29 @@ fn batched_curlp_input_243() {
 }
 
 #[test]
-fn batched_curlp_input_8019() {
+fn batched_curlp_t5b1_input_243() {
```

Comment:
I know it's only tests but still, it bothers me a bit that only one type differs between both impls.
Can we maybe do a function generic over a trit encoding and 2 tests function that just contain a call to the other one ?

Reply:
sure